app/src/main/res/layout/activity_allapps.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:orientation="vertical"
+    android:layout_margin="6dp"
+    android:background="@drawable/round_shape"
+    >
+
+    <SearchView
+        android:id="@+id/search_view"
+        android:theme="@style/LauncherTheme.SearchView"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:queryHint="@string/search_hint"
+        android:focusable="false"
+        android:iconifiedByDefault="false"
+        android:searchIcon="@null"
```

Comment:
add the `ic_search_black_24dp` image here

Reply:
I removed the icon under instructions from @martinhoaragao who said we should have a simplistic for this. I can add the icon back easily, but I'd like a final decision on this so I don't keep going back and forth with the design. My opinion is the icon is unnecessary, and this design is good as it is.

Reply:
If that's so I don't agree with @martinhoaragao and I think that should have a *search icon* so the bar won't go unnoticed. It should have a icon, but if I'm the only one supporting this change, go ahead, I'm not a regular user :stuck_out_tongue_closed_eyes:

Reply:
We might as well get @joaofcosta opinion on this also.

Reply:
![image](https://user-images.githubusercontent.com/17356700/35583203-09f9e78c-05e9-11e8-9419-936bd09005f0.png)
Should there be a magnifying glass icon before the 'Search apps...' or not @martinhoaragao @joaofcosta?

Reply:
I might be going mad, which I wouldn't be surprised about, but I seem to remember pointing out yesterday that the bar should have a search icon. Yet I didn't push the issue as it was not the most worrisome problem.

In fact, I checked my comment on Trello where I mention a more simplistic approach instead of @joaofcosta 's mockup 
> a simple line under the text in gray/black, and the icon on the right in the same colour.
I'm not mad yet! 😜